packages/insured-bridge-relayer/src/RelayerConfig.ts

```diff
@@ -89,6 +85,9 @@ export class RelayerConfig {
     if (!BRIDGE_ADMIN_ADDRESS) throw new Error("BRIDGE_ADMIN_ADDRESS required");
     this.bridgeAdmin = toChecksumAddress(BRIDGE_ADMIN_ADDRESS);
 
+    if (!RATE_MODEL_ADDRESS) throw new Error("RATE_MODEL_ADDRESS required");
```

Comment:
why? can we not get it from the 1.json file using getAddress?

I guess though that this is consistent with the `BRIDGE_ADMIN_ADDRESS` env which requires that this is specified. 

perhaps a future PR can remove this requirement and we place logic in the `index` to fetch the address from the known addresses if this env is not provided.